lib/json_schema/validator.rb

```diff
@@ -249,6 +249,12 @@ def validate_items(schema, data, errors, path)
           false
         else
           valid = true
+          if data.size > schema.items.count && schema.additional_items.is_a?(Schema)
+            (schema.items.count..data.count-1).each do |i|
+              valid = strict_and valid,
+                validate_data(schema.additional_items, data[i], errors, path + [i])
+            end
+					end
```

Comment:
Whoops! I was trying to stick to your space indentation convention, but i usually indent with tabs and have my editor set up to show two space tabs, so it all looked the same in my editor. I meant to convert all tabs to spaces before committing but obviously missed a spot.